README.md

```diff
@@ -76,7 +76,9 @@ described above.
 We provide a CMake build script. This is primarily targeted at Windows users,
 but it can be used in other circumstances where the Autotools script does not
 work.
-
+    
+    $ mkdir build && cd build
+    $ cmake ..
     $ cmake --build .
     $ ctest -V .
```

Comment:
Does this work for you? It fails for me. I think we might need to have cmake copy the test databases to the build directory or something similar.

Reply:
Ah, I see. I must have been on an old version of the branch or something.